src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -47,7 +47,7 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options, IEnu
             _optionsReloadToken = _options.OnChange(ReloadLoggerOptions);
 
             _messageQueue = new ConsoleLoggerProcessor();
-            if (DoesConsoleSupportAnsi())
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || DoesConsoleSupportAnsi())
```

Comment:
@buyaa-n @jeffhandley - here is another scenario where https://github.com/dotnet/runtime/issues/44922 would help.

Reply:
@MaximLipnin could you add `// TODO update when https://github.com/dotnet/runtime/issues/44922 implemented `

Reply:
Updated, thanks